### ISSUE TITLE
Modification of library to work with Arduino Uno (16MHz)

### ISSUE THREAD

**Issue by rocksyne** [LieBtrau/arduino-as3933#3]
Dear Christoph,

Greetings. Regarding the use of this great library, I read in one of the opened issues that you only tested it on 12MHz board. What I intend to do is to read data from the antenna reception into the MCU. I have been trying to modify the code to adapt to my board but I am unable to tune the antenna. The best frequency I am getting is [Frequency: 2057
] and subsequently:

Testing bit: 4
	Frequency: 2057
Testing bit: 3
	Frequency: 3
Testing bit: 2
	Frequency: 3
Testing bit: 1
	Frequency: 3
Testing bit: 0
	Frequency: 3
Calibration for antenna 1 fails

Can you please recommend to me a solution approach? Thank you for your time and have a blessed day.

Kind regards,
Rockson.

**Comment by LieBtrau** [comment]
Hi Rockson,
For the autocalibration steps, the freqcount-library is used, which was designed for 16MHz Arduino uno.  I think the autocalibration steps should work without modifications.
To be honest, I spent quite some time adjusting my hardware to make this work.  The parallel capacitors should be matched to your coil for parallel resonance on 125kHz.  I think the problem was caused by my hardware.  The parallel caps should be as close as possible to the coils.  You'd better use the design of the evaluation kit as a start.
